### PR TITLE
[ALLUXIO-2398]Create document for Running-Hive-with-Alluxio

### PR DESCRIPTION
**Pull request by TeddyBear1314**
https://alluxio.atlassian.net/browse/ALLUXIO-2398
I write a doc about Running Hive With Alluxio. Should Use Running Hive on Alluxio？
I consider that Hive is running on Hadoop MapReduce,so I use with Alluxio.
If you guys have better ideas, please let me know!
And If I need to point out the version of Alluxio to use with Hive, since the current released versions don't support 'setMode'.

### CODE REVIEW

File: docs/en/Running-Hive-with-Alluxio.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+layout: global
+title: Running Apache Hive with Alluxio
+nickname: Apache Hive
+group: Frameworks
+priority: 2
+---
+
+This guide describes how to run [Apache Hive](http://hive.apache.org/) with Alluxio, so
+that you can easily store Hive tables into Alluxio at various storage level.
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: "into Alluxio at various storage level" -> "in Alluxio's tiered storage."

File: docs/en/Running-Hive-with-Alluxio.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+[Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+
+Please [Download Hive](http://hive.apache.org/downloads.html).
+
+# Configuration
+
+Apache Hive allows you to use Alluxio through a generic file system wrapper for the Hadoop file system.
+Therefore, the configuration of Alluxio is done mostly in Hive and its under computing frameworks.
+
+#### Set property in `hive-site.xml`
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: "Set property in" -> "Configure"

File: docs/en/Running-Hive-with-Alluxio.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+<property>
+   <name>fs.defaultFS</name>
+   <value>alluxio://<master_hostname>:19998</value>
+</property>
+```
+
+#### Set the rest properties in Apache Hadoop MapReduce
+
+If you run Hive on Apache Hadoop MapReduce, Hive can read configirations from Hadoop configuration files. In addition,
+Hive's Hadoop jobs will store its immediate results on Alluxio.
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: Did you mean "immediate" -> "intermediate"?

Also, it should be "in Alluxio".

File: docs/en/Running-Hive-with-Alluxio.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+You need to add the following property to `hive-site.xml` in your Hive installation `conf` directory
+
+```xml
+<property>
+   <name>fs.defaultFS</name>
+   <value>alluxio://<master_hostname>:19998</value>
+</property>
+```
+
+#### Set the rest properties in Apache Hadoop MapReduce
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: Configure Hadoop MapReduce

File: docs/en/Running-Hive-with-Alluxio.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+Distributing the Alluxio Client Jar in Hadoop MapReduce:
+
+- You must place the client jar
+`alluxio-core-client-{{site.ALLUXIO_RELEASED_VERSION}}-jar-with-dependencies.jar`
+(located in the `/<PATH_TO_ALLUXIO>/core/client/target/` directory), in the `$HADOOP_HOME/lib`
+(may be `$HADOOP_HOME/share/hadoop/common/lib` for different versions of Hadoop) directory of every
+MapReduce node, and restart all of the NodeManagers.
+
+# Distribute the Alluxio Client jar
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: how is this section different from the previous section about putting the jar in `$HADOOP_HOME/lib`?

File: docs/en/Running-Hive-with-Alluxio.md (hunk 4 above)
Comment (TeddyBear1314) [inline docs/en/Running-Hive-with-Alluxio.md]: Got it! I'll change it soon!

File: docs/en/Running-Hive-with-Alluxio.md (hunk 5 above)
Comment (TeddyBear1314) [inline docs/en/Running-Hive-with-Alluxio.md]: There is no difference!  I was considering to refer the ’Configure Hadoop‘ to the previous section, but I want this doc to be standalone. So is this idea sensible?
By the way, I want to change the order of the '../lib', in my Hadoop-2.7.1, `$HADOOP_HOME/share/hadoop/common/lib` works.

File: docs/en/Running-Hive-with-Alluxio.md (hunk 3 above)
Comment (TeddyBear1314) [inline docs/en/Running-Hive-with-Alluxio.md]: Oops! Poor English of me!

**Comment by apc999** [comment]
@TeddyBear1314  thanks for the PR! it looks good to me. On the other hand, could you make this PR targeting master branch rather than 1.3 branch. The reason is that this doc will not work with 1.3.0 (but works with 1.3.0-SNAPSHOT or later). So it could be misleading if the doc sits in 1.3 doc.

**Comment by apc999** [comment]
Sorry I was confused and thought this going to branch 1.3. In this case that it is going to mater, LGTM .

**Pull request merged**